intentv1/intent.ridl

```diff
@@ -0,0 +1,103 @@
+webrpc = v1
+
+name = sequence-waas-intents
+version = v0.1.0
+
+struct Signature
+  - sessionId: string
+  - signature: string
+
+# no way to generate string enums
+# enum IntentName: uint8
+#  - openSession
+#  - closeSession
+#  - validateSession
+#  - finishValidateSession
+#  - listSessions
+#  - getSession
+#  - sign
+#  - transaction
+
+struct Intent
+  - version: string
+  - name: string
+  - expires: uint64
```

Comment:
can we rename to "expiresAt: uint64" and also, "issuedAt: uint64" ..? its just more consistent with the names used in JWT.. just feels more familiar

Reply:
done